README.md

```diff
@@ -59,7 +59,25 @@ Read the [Roadmap](https://pingcap.com/docs/ROADMAP).
 
 ## Quick start
 
-Read the [Quick Start Guide](https://pingcap.com/docs/QUICKSTART), which includes deployment methods using Ansible, Docker, and Kubernetes.
+- Install [TiUP](https://tiup.io)
+
+    ```shell script
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh 
+    ```
+  
+- Run a TiDB cluster (the latest stable version)
+
+    ```shell script
+    tiup playground
+    ```
+
+- Run a nightly TiDB cluster (the nightly built version)
+    
+    ```shell script
+    tiup playground nightly
+    ```
+
+See [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/) for other quick start methods.
```

Comment:
```suggestion
See [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/) for other quick start ways.
```
How about this?

Reply:
em, see https://github.com/pingcap/tidb/pull/17188#discussion_r424844436